assets/js/main.js

```diff
@@ -1,29 +1,38 @@
-
 (function () {
-
 	// add more files here if needed
-        var files = ['javascript.json', 'text-editors.json', 'jquery.json', 'node.json']
+    var files = ['javascript.json', 'node.json', 'jquery.json', 'text-editors.json']
```

Comment:
@arthurvr gah.  text editor issue w tabs vs. spaces.  it is lined up technically. will address it.